falco/values.yaml

```diff
@@ -51,100 +59,158 @@ resources:
+  readinessProbe:
+    initialDelaySeconds: 30
+    timeoutSeconds: 5
+    periodSeconds: 15
+extra:
+  # Extra environment variables that will be pass onto Falco containers
   env: {}
+  # Extra command-line arguments
+  args: []
+  # Additional initContainers for Falco pods
+  initContainers: []
+  # Extra volumes to Falco DaemonSet/Deployment
+  volumes: []
+  # Extra volumeMounts to Falco containers
+  volumeMounts: []
+
+#########################
+# Scenario requirements #
+#########################
+
+controller:
+  kind: daemonset
+  daemonset:
+    # Perform rolling updates by default in the DaemonSet agent
+    # ref: https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/
+    updateStrategy:
+      # You can also customize maxUnavailable or minReadySeconds if you
+      # need it
+      type: RollingUpdate
+  deployment:
+    replicas: 1
+
+# Network services configuration (scenario requirement)
+services:
+  # Example configuration for the "k8sauditlog" plugin
+  # - name: k8saudit-webhook
+  #   type: NodePort
+  #   ports:
+  #     - port: 9765 # See plugin open_params
+  #       nodePort: 30007
+  #       protocol: TCP
+
+# Driver settings (scenario requirement)
+driver:
+  enabled: true
+  kind: ebpf
+  ebpf:
+    path:
+    # Needed to enable eBPF JIT at runtime for performance reasons.
+    # Can be skipped if eBPF JIT is enabled from outside the container
+    hostNetwork: true
+    # Constrain Falco with capabilities instead of running a privileged container.
+    # This option is only supported with the eBPF driver and a kernel >= 5.8.
+    # Ensure the eBPF driver is enabled (i.e., setting the `driver.kind` option to `ebpf`).
+    leastPrivileged: false
+  loader:
+    enabled: true
+    initContainer:
+      enabled: true
+      image:
+        pullPolicy: IfNotPresent
+        registry: docker.io
+        repository: falcosecurity/falco-driver-loader
+        # Overrides the image tag whose default is the chart appVersion.
+        tag: ""
+
+collectors:
+  enabled: true
 
-  ## Add aditional pod annotations on pods created by DaemonSet
-  podAnnotations: {}
-
-# Additional labels to add to the pods:
-# podLabels:
-#   key: value
-podLabels: {}
-
-# If is behind a proxy you can set the proxy server
-proxy:
-  httpProxy:
-  httpsProxy:
-  noProxy:
-
-# Set daemonset timezone
-timezone:
+  docker:
+    enabled: true
+    socket: /var/run/docker.sock
 
-# Set daemonset priorityClassName
-priorityClassName:
+  containerd:
+    enabled: true
+    socket: /run/containerd/containerd.sock
 
-ebpf:
-  # Enable eBPF support for Falco
-  enabled: false
-  path:
+  crio:
+    enabled: true
+    socket: /run/crio/crio.sock
 
-  settings:
-    # Needed to enable eBPF JIT at runtime for performance reasons.
-    # Can be skipped if eBPF JIT is enabled from outside the container
-    hostNetwork: true
+  kubernetes:
+    # Enables Kubernetes meta data collection via a connection to the Kubernetes API server.
+    enabled: true
```

Comment:
TODO: add a note that Kubernetes metadata will be partially available even with this disabled.

Reply:
done